saleor/tests/e2e/checkout/test_unlogged_customer_should_not_be_able_to_order_product_quantity_greater_than_stock.py

```diff
@@ -84,7 +84,7 @@ def prepare_product(
 
 
 @pytest.mark.e2e
-def test_unlogged_customer_cannot_buy_product_in_quantity_grater_than_stock(
+def test_logged_customer_cannot_buy_product_in_quantity_grater_than_stock(
```

Comment:
In the new test name, you set `logged_customer` but the test use `e2e_not_logged_api_client`.